### DISCUSSION TITLE
Welcome to Oscar.jl Discussions!

### DISCUSSION THREAD

**Discussion by fingolfin** [oscar-system/Oscar.jl#499] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gustavo2121** [comment]
please sir,

could you help me solve this problem?
I am using ubundu interface on my window 10
![oscar](https://user-images.githubusercontent.com/91778451/150490528-8d455184-a6a0-452c-888f-025aefa0378f.PNG)

**Comment by thofma** [comment]
We require julia >= 1.6 and it seems that you did not use the official julia version available from https://julialang.org/downloads/.

**Reply by gustavo2121** [reply]
Thank you very much for your reply. and sorry for the beginner's question. After I have choosen the **Generic Linux on x86**. Do you then advise to use **Linux and FreeBSD** to setup Julia or I should use the **Windows** setting when I am here https://julialang.org/downloads/platform/#linux_and_freebsd?  

Thank you in advance for your reply

**Reply by fingolfin** [reply]
As described on https://oscar.computeralgebra.de/install/, you need to
- install WSL (Windows Subsystem for Linux)
- install Julia for Linux (not for Windows!) inside WSL

**Comment by thofma** [comment]
For the ubuntu interface inside Windows 10, you have to install julia by following the steps under https://julialang.org/downloads/platform/#linux_and_freebsd.

**Reply by gustavo2121** [reply]
Thank you very much for your reply thofma.
let me try it again.

**Reply by gustavo2121** [reply]
Sir, 

please help again I am afraid i don't know how to create the symbolic link:
This is the information on my system path  C:\Users\JohnDoe\AppData\Local\Programs\Julia 1.7.1\bin
I saw that I have to run this: ln -s /path/to/original /path/to/link
now I don't really know what to put 
![ubundu](https://user-images.githubusercontent.com/91778451/150527499-040beb27-c76b-4e54-9e81-55f951343912.PNG)

This is where I am now. Sorry if my questions sound stupid. I am new I programming and I really want to learn this language.
Thank you

**Comment by thofma** [comment]
You don't need the symlink. If this is the julia within the WSL, then `]add Oscar` will install Oscar.

**Comment by fingolfin** [comment]
Instead of commenting on this announcement thread, please open a separate discussion or issue. I.e, go to https://github.com/oscar-system/Oscar.jl/discussions and click "New Discussion".  I will now lock this now announcement for replies.
